include/simdjson/jsonioutil.h

```diff
@@ -17,16 +17,18 @@ namespace simdjson {
 // throws exceptions in case of failure
 // first element of the pair is a string (null terminated)
 // whereas the second element is the length.
-// caller is responsible to free (aligned_free((void*)result.data())))
 //
 // throws an exception if the file cannot be opened, use try/catch
+//
+//      padded_string p{} ;
 //      try {
 //        p = get_corpus(filename);
 //      } catch (const std::exception& e) {
-//        aligned_free((void*)p.data());
+//          p.~padded_string() ;
```

Comment:
Calling the destructor in such a way almost ensures that the destructor will be called twice.

Reply:
already removed